include/cutlass/epilogue/dispatch_policy.hpp

```diff
@@ -131,13 +131,15 @@ template<
   int StagesC_,
   int StagesD_,
   int FragmentSize_,
-  bool ReuseSmemC_
+  bool ReuseSmemC_,
+  bool EnableNullptrD_ = true
```

Comment:
See comment above about using the element D to detect this instead of adding a new parameter here.